DOC : Fixes the issue #2611 by explicitly mentioning the default values (#2891)
zarr-developers/zarr-python: docs/user-guide/config.rst

Configuration options include the following:

- ``default_zarr_version``  

>   Sets the default Zarr format version.

Reviewer: ```suggestion
  Sets the default Zarr format version. The options are ``2`` or ``3``.
```

  - **Default:** ``3`` 

- **array.order**  
  Defines the default memory layout for arrays.  

>   - ``'C'`` → Row-major (default, like NumPy)  

Reviewer: ```suggestion
  - ``'C'`` → Row-major (The default value used for NumPy arrays)  
```
numpy arrays can also be F-ordered.

  - ``'F'`` → Column-major (like Fortran)  
  - **Default:** ``'C'``  
 

- Default filters, serializers and compressors, e.g. ``array.v3_default_filters``, ``array.v3_default_serializer``, ``array.v3_default_compressors``, ``array.v2_default_filters`` and ``array.v2_default_compressor``

> - ``array.write_empty_chunks``
>   Determines whether empty chunks (filled with default values) are written to storage.
> 
>   - ``False`` → Empty chunks are **not written** (saves space).  
>   - ``True`` → Empty chunks are explicitly stored.  
>   - **Default:** ``False``  

Reviewer: ```suggestion
- ``array.write_empty_chunks``
  Determines whether empty chunks (filled with default values) are written to storage. Setting this to ``False`` can reduce the number of write operations and objects created when writing arrays with large empty regions.

  - ``False`` → Empty chunks are **not written**.  
  - ``True`` → Empty chunks are explicitly stored.  
  - **Default:** ``False``  
```
